docs/environment_variables.rst

```diff
@@ -0,0 +1,34 @@
+.. meta::
+   :description: HSA runtime implementation
+   :keywords: ROCR, ROCm, library, tool, runtime
+
+.. _environment-variables:
+
+Environment variables
+========================
+
+The following table lists the useful environment variables.
```

Comment:
The term "useful" should be removed. 
it means that other variables are useless. 

Are these the only variables? if not then say something like, 
The following table lists the most often used variables.

Reply:
updated.